.github/workflows/testing.yml

```diff
@@ -223,7 +223,7 @@ jobs:
           # There are tarantool releases on which AppleClang
           # complains about the problem that was fixed later in
           # https://github.com/tarantool/tarantool/commit/7e8688ff8885cc7813d12225e03694eb8886de29
-          cmake . -DCMAKE_BUILD_TYPE=RelWithDebInfo
+          cmake . -DCMAKE_BUILD_TYPE=RelWithDebInfo -DOPENSSL_ROOT_DIR=/usr/local/opt/openssl@1.1 -DOPENSSL_LIBRARIES=/usr/local/opt/openssl@1.1/lib
```

Comment:
I would leave a comment why explicit options related to openssl is passed there: just like build type is commented above. You can mention that this should be revisited after fixing of https://github.com/tarantool/tarantool/issues/6477.

Reply:
Sure, the comment is left there in the latests update.